Android/Io.OpenCensus/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
This license needs to follow the format as seen here: https://github.com/xamarin/XamarinComponents/blob/master/Android/Guava/License.md

Reply:
Done